static/js/components/search/CustomPaginationDisplay_test.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+import sinon from 'sinon';
+
+import CustomPaginationDisplay from './CustomPaginationDisplay';
+
+describe('CustomPaginationDisplay', () => {
+  let toggleItemStub = sinon.stub();
```

Comment:
with this stub you should make sure you're calling `toggleItemStub.reset()` after each test - otherwise the value for `called` isn't reset to `false` before the test runs. you could do this in a `beforeEach` or `afterEach` callback.